src/aind_data_schema_models/mouse_ccf.py

```diff
@@ -0,0 +1,55 @@
+"""Module for CCF Area definitions"""
+
+from typing import List
+from typing_extensions import Annotated
+from importlib_resources import files
+from pydantic import ConfigDict, Field
+
+from aind_data_schema_models.pid_names import BaseName
+from aind_data_schema_models.utils import create_literal_class, read_csv
+
+
+def map_pid(value: str, record: dict):
+    """Replace the parent_structure_id with an int"""
+    if not value:
+        record["parent_id"] = Annotated[None, Field(None)]
+    else:
+        record["parent_id"] = Annotated[int, Field(int(float(value)))]
+
+
+def map_structure_list(value: str, record: dict):
+    """Replace the structure list with a List[int]"""
+    record["structure_id_path"] = Annotated[List[int],
+                                            Field(list(map(int, value.strip('/').split('/'))))]
+
+
+class CCFStructureModel(BaseName):
```

Comment:
I'm not sure we want all of this in the model for the schema. I think we only want acronym, name, and id.

Reply:
and probably registry and registry_identifier (instead of id)

Reply:
Okay these are changed

Reply:
one question is do we want this to be specific to CCF or generic to any atlas. 
WE will be using the CCF for most everything we do. If we want this to be generic for other users, it could be designed as something more flexible. And we could call it `BrainStructureModel` or something like that? @dyf what do you think?